### PR TITLE
bpo-40173: Remove redundant import module of import_fresh_module()

### PR DESCRIPTION
**Pull request by shihai1991**
https://bugs.python.org/issue40173

### CODE REVIEW

**Review by iritkatriel — COMMENTED** [review]
This PR needs a unit test.

**Comment by shihai1991** [comment]
> This PR needs a unit test.

Thanks for your comment. I have forget this PR's detail. test.support need many test cases. 
After we make sure this PR can fix the issue, I can add some test cases.

**Comment by iritkatriel** [comment]
Up to you, but I think it will be easier for reviewers to see what issue the PR is trying to fix if it included tests.

File: Lib/test/support/__init__.py [hunk 1]
```diff
@@ -240,8 +240,10 @@ def _save_and_remove_module(name, orig_modules):
     """
     # try to import the module and raise an error if it can't be imported
     if name not in sys.modules:
+        orig_modules = sys.modules.copy()
         __import__(name)
-        del sys.modules[name]
+        sys.modules.clear()
+        sys.modules.update(orig_modules)
```
Comment (serhiy-storchaka) [inline Lib/test/support/__init__.py:246]: I considered this option, but some modules can not support re-importing. So it is safer to restore only explicitly mentioned modules and their submodules.
See #28654.

Comment (shihai1991) [reply Lib/test/support/__init__.py:246]: Oh, sorry, Serhiy. I missed your PR :(

**Pull request closed without merging**